CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/CollectionRequestHandler.swift

```diff
@@ -0,0 +1,142 @@
+//RequestHandler.swift
+//CBLTestServer-iOS
+
+import Foundation
+import CouchbaseLiteSwift
+
+public class CollectionRequestHandler {
+    public static let VOID: String? = nil
+    fileprivate var _pushPullReplListener:NSObjectProtocol?
+    
+    public func handleRequest(method: String, args: Args) throws -> Any? {
+        switch method {
+        case "collection_defaultCollection":
+            let database: Database = (args.get(name:"database"))!
+            let collection: Collection = try (database.defaultCollection())!
```

Comment:
this will crash in case default collection is empty. 

may be something like this? so the result is optional.
```suggestion
            let collection: Collection? = try database.defaultCollection()
```